File: Anul_2/Semestrul_2/Medii de proiectare si programare/mpp-proiect-csharp-andrea-muresan/TravelAgencyProject/Model/dto/DtoUtils.cs
namespace Model.dto
{
    public class DtoUtils
    {
        public static User GetFromDTO(UserDto userDto)
        {
            string username = userDto.Username;
            string password = userDto.Password;
            return new User(username, password);
        }

        public static UserDto getDTO(User user)
        {
            string username = user.Username;
            string password = user.Password;
            return new UserDto(username, password);
        }
            
    }
}

File: travelAgency/Model/src/main/java/app/model/dto/DtoUtils.java
package app.model.dto;

import app.model.User;

public class DtoUtils {
    public static User getFromDTO(UserDto userDto){
        String username = userDto.getUsername();
        String password = userDto.getPassword();
        return new User(username, password);
    }
    public static UserDto getDTO(User user){
        String username = user.getUsername();
        String pass = user.getPassword();
        return new UserDto(username, pass);
    }
}